### ISSUE TITLE
Transition Tests pass in local reference tests but fail for retesteth

### ISSUE THREAD

**Issue by shemnon** [hyperledger/besu#1211]
Several tests that pass in the gradle based reference test execution fail when run in retesteth.  From retesteth.etherdevops.io the current list is:
```
(bcEIP158ToByzantium/ByzantiumTransition_EIP158ToByzantiumAt5, fork: EIP158ToByzantiumAt5, block: 4)
(bcByzantiumToConstantinopleFix/ConstantinopleTransition_ByzantiumToConstantinopleFixAt5, fork: ByzantiumToConstantinopleFixAt5, block: 4)
(bcHomesteadToDao/DaoTransactions_EmptyTransactionAndForkBlocksAhead_HomesteadToDaoAt5, fork: HomesteadToDaoAt5, block: 5)
(bcHomesteadToDao/DaoTransactions_HomesteadToDaoAt5, fork: HomesteadToDaoAt5, block: 9)
(bcHomesteadToDao/DaoTransactions_UncleExtradata_HomesteadToDaoAt5, fork: HomesteadToDaoAt5, block: 5)
(bcHomesteadToEIP150/EIP150Transition_HomesteadToEIP150At5, fork: HomesteadToEIP150At5, block: 4)
(bcFrontierToHomestead/HomesteadOverrideFrontier_FrontierToHomesteadAt5, fork: FrontierToHomesteadAt5, block: 4)
(bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain2_FrontierToHomesteadAt5, fork: FrontierToHomesteadAt5, block: 4)
(bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain_FrontierToHomesteadAt5, fork: FrontierToHomesteadAt5, block: 4)
```

**Comment by winsvega** [comment]
Today's build has passed the transition tests

**Comment by shemnon** [comment]
Fixed by #1212
